src/modules/transformer/transformer.controller.spec.ts

```diff
@@ -1,20 +1,148 @@
+import { TransformerService } from './transformer.service';
+import { TransformerController } from './transformer.controller';
 import { Test, TestingModule } from '@nestjs/testing';
-import { ConversationLogicController } from '../conversation-logic/conversation-logic.controller';
-import { ConversationLogicService } from '../conversation-logic/conversation-logic.service';
+import { AddOwnerInfoInterceptor } from '../../interceptors/addOwnerInfo.interceptor';
+import { AddAdminHeaderInterceptor } from '../../interceptors/addAdminHeader.interceptor';
+import { createMock } from '@golevelup/ts-jest';
 
-describe.skip('ConversationLogicController', () => {
-  let controller: ConversationLogicController;
+describe('TransformerController', () => {
+  let controller: TransformerController;
+
+  const mockTransformerData = {
+    id: '123',
+    createdAt: 1000,
+    updatedAt: 1000,
+    name: 'mockTransformerService',
+    service: { create: 'create' },
+    tags: ['tag1', 'tag2'],
+    config: {
+      value: '123',
+    },
+  };
+
+  const mockTransformerService = {
+    create: jest.fn((data) => {
```

Comment:
Hey @1010varun this PR is supposed to contain test fixes for `user-segment.service` but it contains changes for `transformer.controller` as well which as addressed on this [PR](https://github.com/samagra-comms/uci-apis/pull/194). Can you clean this PR up to only contain the necessary changes after rebasing from the latest `develop` branch?